yarn.lock

```diff
@@ -9264,6 +9294,16 @@ yocto-queue@^1.0.0:
   resolved "https://registry.npmjs.org/yocto-queue/-/yocto-queue-1.0.0.tgz"
   integrity sha512-9bnSc/HEW2uRy67wc+T8UwauLuPJVn28jb+GtJY16iiKWyvmYJRXVT4UamsAEGQfPohgr2q4Tq0sQbQlxTfi1g==
 
+yup@^1.3.2:
```

Comment:
bit weird that yup is still in the lockfile? can you try a `yarn install` and commit the latest lockfile that is output.

Reply:
fixed